Resources/Prototypes/Loadouts/Jobs/Security/uncategorized.yml

```
@@ -750,3 +750,37 @@
         - ZavodskoiInterstellar
   items:
     - ClothingUniformJumpsuitZavodCadet
+
+    - type: loadout
+  id: LoadoutSecurityJumpsuitEOC
```

YAMLlint (1.35.1)

[error] duplication of key "id" in mapping

(key-duplicates)